src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/DocumentResource.kt

```diff
@@ -102,7 +102,7 @@ class DocumentResource(private val documentService: DocumentService) {
     @Schema(description = "The booking id") @PathVariable bookingId: Long,
     @RequestParam(value = "templateName", required = true)
     @Parameter(
-      description = "The unique name of the template used for a document",
+      description = "The unique name of the template used for a document - this is a repeatable request parameter",
```

Comment:
The swagger docs show this as an array of string values - but added extra info in description to be more explicit - after review